### DISCUSSION TITLE
Curated collections

### DISCUSSION THREAD

**Discussion by erlend-sh** [omnivore-app/omnivore#1845] [Ideas]
This might be an extension of Clubs or its own thing. The main difference from Clubs as currently envisioned is that these collections are meant to be shared globally.

In short, I’d like the ability to create curated lists of roughly the same kind as what you’ll find on IMDB:
https://www.imdb.com/list/ls055592025/

But rather than writing a custom title for a collection, I think it’d be more complementary to the existing system to just use [(global) labels](https://github.com/omnivore-app/omnivore/issues/1842), or combinations of labels. E.g.

#### burnout + open-source

1. https://mikemcquaid.com/open-source-maintainers-owe-you-nothing/
2. https://nolanlawson.com/2017/03/05/what-it-feels-like-to-be-an-open-source-maintainer/
3. https://www.wired.com/story/open-source-coders-few-tired/
4. https://writing.jan.io/2017/03/06/sustainable-open-source-the-maintainers-perspective-or-how-i-learned-to-stop-caring-and-love-open-source.html
5. https://medium.com/@mikeal/time-to-leave-a68294ccb2af#.p8ss5xeqz

The key difference from having a bunch of articles with a certain label is that a Collection has a ranking, with (1) being the top recommendation. That way you’ll have two key data points that can be used to arrive at a global list of the top3/top5/etc. #burnout+#opensource articles.

I’m making this a discussion because there’s still some details to settle about how to design this. My current thinking is that for any group of articles with a certain label, you can optionally designate 3-5 articles of that label with a ranking, whereas the rest would be unranked.

**Comment by erlend-sh** [comment]
`#protocols`

https://efdn.notion.site/0314a1800b774258a8e6197487c479bc?v=15fb6b39a490413cbba3b7f78ad67c2b

**Comment by ishowman** [comment]
omnivore is a good collection tool, and I think it will be greate if I could create list with a theme and add some collected tool to it, so some awesome article could be organized and shared.

**Comment by jacksonh** [comment]
Thanks, this does tie into some stuff we'd like to try. Do you envision these being updated automatically when you label something?

**Comment by uogbuji** [comment]
Any updates on this? Basically I'd love to use Omnivore in context of a local AI interest group, curated by multiple folks and shared with anyone who finds the link. Anything like this coming any time soon?
